### DISCUSSION TITLE
Welcome to revm Discussions!

### DISCUSSION THREAD

**Discussion by rakita** [bluealloy/revm#1462] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chifox-tw** [comment]
Hello everyone! I am junior blockchain engineer from Taiwan. Mainly skilled in Solidity / Typescript 
I am using my remaining work time to learn Rust, and finding a opportunity to join open-source project contribution
